react/test/components/FollowButtonSpec.js

```diff
@@ -0,0 +1,25 @@
+import FollowButton from '../../src/components/FollowButton';
+
+describe('FollowButton', ()=> {
+  let wrapper;
+
+    beforeEach(() => {
+      wrapper = mount(
+        <FollowButton
+        />
+      )
+    });
+
+  it('should return true', () => {
+    expect(true).toEqual(true)
+  })
+
+  it('should render a button tag', () => {
+    expect(wrapper.find("button")).toBePresent()
+  })
+
+  it('should render a button tag with text', () => {
+    expect(wrapper.find("button").text()).toEqual("Follow Button")
+  })
+
+})
```

Comment:
Because your follow button isnt actually working yet, you should not have a test for it. Save till later/ delete